lib/token.h

```diff
@@ -441,6 +441,7 @@ class CPPCHECKLIB Token {
     void isSigned(const bool sign) {
         setFlag(fIsSigned, sign);
     }
+    // cppcheck-suppress unusedFunction
```

Comment:
Is this flag used at all?

Reply:
I have no idea. I will leave these (and others) in for now and re-visit those suppressions in a few months and get rid of them if they are still not used. I don't want to blow up the scope of this PR or drag it out.

Reply:
It was added in 66d145ba1c338bfa3d36d4f602b02772ddd9dc1d and its usage was removed in 5216f904e70da6585f858531d15b8c292944d12c. So I guess we can drop this soon.